src/main/resources/file-download-context.xml

```diff
@@ -71,6 +71,8 @@
 		<value>ComplexPortal</value>
 		<value>ComplexPortalSARS</value>
 		<value>ComplexPortalSARSCOV2</value>
+		<value>IUPHAR</value>
```

Comment:
You might need to do a search and replace for "IUPHAR" to "Guide To Pharmacology" in the project since it's their new name and the reference database in gk_central will be renamed very soon.

Reply:
Ugh... I was hoping to leave that for Release 77. :/

Reply:
Does the program use this name to try to grab the reference database instance?  If not, you could leave it.  If so, it may still work because "IUPHAR" will be one of the values in the "name" attribute but it won't be the first one or the "displayName"

Reply:
IUPHAR is not in reference-databases.xml, so it doesn't really matter what name is used in the Spring context files. I think changing the Spring identifier "IUPHAR" to "GtP" (or whatever) can wait until later (i.e. when I'm less busy).